src/main/webapp/WEB-INF/tags/instructor/results/filterEditModal.tag

```diff
@@ -0,0 +1,120 @@
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/functions" prefix="fn" %>
+<%@ tag description="instructorFeedbackResultsTop - Filter Panel Edit Modal" %>
```

Comment:
The tag description should still be the very first line. I meant that the tag import should not come between core and functions. Just move the description to the first line and all will be fine.